Formula/k/kubehound.rb

```diff
@@ -0,0 +1,33 @@
+class Kubehound < Formula
+  desc "Tool for building Kubernetes attack paths"
+  homepage "https://kubehound.io"
+  url "https://github.com/DataDog/KubeHound/archive/refs/tags/v1.3.3.tar.gz"
+  sha256 "5ee3ca8621d4611e682ed2325bb12b87d6f00c4f7717195d5f43856f5d9b7917"
+  license "Apache-2.0"
+
+  depends_on "go" => [:build, :test]
+
+  def install
+    goos = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOOS").chomp
+    goarch = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOARCH").chomp
+
+    ldflags = %W[
+      -s -w
+      -X github.com/DataDog/KubeHound/pkg/config.BuildVersion=v#{version}
+      -X github.com/DataDog/KubeHound/pkg/config.BuildBranch=main
+      -X github.com/DataDog/KubeHound/pkg/config.BuildOs=#{goos}
+      -X github.com/DataDog/KubeHound/pkg/config.BuildArch=#{goarch}
+    ]
+    system "go", "build", *std_go_args(ldflags:), "./cmd/kubehound/"
+  end
+
+  test do
+    # Check that the binary is working
+    assert_match "kubehound version: v#{version}", shell_output("#{bin}/kubehound version")
+
+    # Docker is required at runtime for some command but test environment does not permit such tests
+    ENV["DOCKER_HOST"] = "unix://#{testpath}/invalid.sock"
```

Comment:
we dont have to set that, I think it would be the same error msg without it

Reply:
I think removing this line won't impact the test in the CI, but it will make the test fail when running locally in a environment that has access to docker, because it will actually manage to connect to the docker daemon

I think it's better to keep it so we have the same behaviour on all environments. I added an additional comment to mention that, if you agree